portfolio.html

```diff
@@ -4,10 +4,99 @@
     <meta charset="UTF-8">
     <title>portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="normalize" type="text/css>" href="normalize.css">
 </head>
+
 <body>
+
     <header>
 
+        <nav class="main-nav">
+            <a href="">Home</a>
+            <a href="#aboutme">About</a>
+            <a href="#portfolio">Portfolio</a>
+            <a href="#contact">Contact info</a>
+        </nav>
     </header>
+
+    <section>
```

Comment:
Consider putting the class mini-profile on the section rather than creating a div

Reply:
Changed